docs/events.md

```diff
@@ -97,6 +98,8 @@ There are plans to provide non-blocking event receivers, so stay tuned.
 
 Exceptions in event receivers are logged, but will not affect the current db transaction.
 That way, you can be sure the transaction will not be rolled-back because of an Exception in an event receiver.
+However, if the `raise_exceptions` argument of the receiver is `True` it will raise any
+exceptions that ocurred in the receiver function and the transaction will be rolled back.
```

Comment:
Same here, max one exception will be raised per receiver.

Reply:
Fixed